### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by CarstenWalther**
* Update merchant documentation to reflect the new code changes
* Update jupyter notebook, so that code examples work with the current code base
* Add a new jupyter notebook, which explains how to build a merchant on top of the merchant base class
* Fixed the abstract base class. Previously it was possible to instantiate the abstract class without getting an error.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,68 +2,50 @@
...
 * Cheapest
-* Second Cheapest
-* Random Third
-  * randomly chooses either the first, second or third position in the price ranking
-* Two Bound 
+  * always undercuts the competitors
+* Two Bound
   * has a minimum and maximum profit margin
   * keeps decreasing the price to maintain the 1st position in the price rank until the minimum bound is reached
   * then set the price back to the maximum margin
```
Comment (Bouncner) [inline README.md]: Do we really use margin, not price?

File: README.md [hunk 2]
```diff
@@ -2,68 +2,50 @@
...
-* Fix Price
-* Maximize Highest Expected Profit
-  * i.e. a data-driven, machine learning merchant using logistic regression
 
-Moreover, the repository contains an SDK to easily build arbitrary merchant behaviors yourself.
+There are three ways to build your own merchant:
+* Use the example merchant and make changes to it (`merchant.py`)
+* [Subclass the merchant base class and extend it](docs/Building%20a%20merchant%20using%20PricewarsMerchant.ipynb)
```
Comment (Bouncner) [inline README.md:16]: Is this notebook still fully up to date?

File: README.md (hunk 1 above)
Comment (CarstenWalther) [reply README.md]: You are right. It's the price

File: README.md (hunk 2 above)
Comment (CarstenWalther) [reply README.md:16]: This notebook is newly created in this PR

**Comment by Bouncner** [comment]
Loving the notebooks!

**Pull request merged**